src/asciilogo.h

```diff
@@ -28,20 +28,20 @@
  */
 
 const char *ascii_logo =
-"                _._                                                  \n"
-"           _.-``__ ''-._                                             \n"
-"      _.-``    `.  `_.  ''-._           Redis %s (%s/%d) %s bit\n"
-"  .-`` .-```.  ```\\/    _.,_ ''-._                                  \n"
-" (    '      ,       .-`  | `,    )     Running in %s mode\n"
-" |`-._`-...-` __...-.``-._|'` _.-'|     Port: %d\n"
-" |    `-._   `._    /     _.-'    |     PID: %ld\n"
-"  `-._    `-._  `-./  _.-'    _.-'                                   \n"
-" |`-._`-._    `-.__.-'    _.-'_.-'|                                  \n"
-" |    `-._`-._        _.-'_.-'    |           https://redis.io       \n"
-"  `-._    `-._`-.__.-'_.-'    _.-'                                   \n"
-" |`-._`-._    `-.__.-'    _.-'_.-'|                                  \n"
-" |    `-._`-._        _.-'_.-'    |                                  \n"
-"  `-._    `-._`-.__.-'_.-'    _.-'                                   \n"
-"      `-._    `-.__.-'    _.-'                                       \n"
-"          `-._        _.-'                                           \n"
-"              `-.__.-'                                               \n\n";
+"                    ___   ___                                                           \n"
+"                    (   ) (   )                                                         \n"
+" ___  ___    .---.   | |   | |   ___     .--.    ___  ___                               \n"
+"(   )(   )  / .-, \\ | |   | |  (   )   /   \\  (   )(   )   Valkey %s (%s/%d) %s bit  \n"
+" | |  | |  (__) ; |  | |   | |  ' /    |  .-. ;  | |  | |                               \n"
+" | |  | |    .'`  |  | |   | |,' /     |  | | |  | |  | |    Running in %s mode         \n"
+" | |  | |   / .'| |  | |   | .  '.     |  |/  |  | '  | |    Port: %d                   \n"
+" | |  | |  | /  | |  | |   | | `. \\    |  ' _.'  '  `-' |    PID: %ld                 \n"
+" ' '  ; '  ; |  ; |  | |   | |  \\ \\    |  .'.-.   `.__. |                            \n"
+" \\  `' /   ' `-'  |  | |   | |   \\ .   '  `-' /   ___ | |                             \n"
+"   '_.'    `.__.'_. (___) (___ ) (___)  `.__.'   (   )' |   https://github.com/valkey-io\n"
```

Comment:
We have this domain name already so let's use it. Currently it's a redirect to this repo but that'll change once we have a website.

```suggestion
"   '_.'    `.__.'_. (___) (___ ) (___)  `.__.'   (   )' |   https://valkey.io\n"
```